script/foreman-debug

```diff
@@ -86,21 +86,24 @@ add_file() {
   MIME=$(file -bi "$FILE" | cut -d\; -f1)
   case $MIME in
         application/x-gzip)
-          zcat "$FILE" | sed -r "$FILTER" > "$DIR$FILE.txt"
-	  touch -c -r "$FILE" "$DIR$FILE.txt"
+          OUTFILE=$(echo "$FILE" | sed s/.gz$//)
```

Comment:
@lzap Thanks, tried % flag to match .gz at the end of the line and it worked great. Updating the PR.